source/blog/2018-11-19-ember-3-6-released.md

```diff
@@ -0,0 +1,207 @@
+
+Wrapping up this final stage we introduce two new events to the router service: `routeWillChange` and `routeDidChange`. They both receive a single `transition` argument which has been expanded to now include `to` and `from` properties. This allows you to understand where the transition is and where it is going. To see some examples of this be sure to read the [events section](https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md#new-events-routewillchange--routedidchange) of the RFC.
+Be sure to have a look at the [RFC itself](https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md) for more info in the router service.
+
+#### Deprecations (6)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.6, please review the [Ember.js 3.6 release page](https://github.com/emberjs/ember.js/releases/tag/v3.6.0).
+
+**new EmberObject (1 of 6)**
+
+We are deprecating usage of `new EmberObject()` to construct instances of `EmberObject` and it's subclasses. This affects all classes that extend from `EmberObject` as well, including user defined classes and Ember classes such as:
+
+* `Component`
+* `Controller`
+* `Service`
+* `Route`
+* `Model`
+
+Instead, you should use `EmberObject.create()` to create new instances of classes that extend from `EmberObject`. If you are using native class syntax instead of `EmberObject.extend()` to define your classes, you can also refactor to _not_ extend from `EmberObject`, and continue to use `new` syntax.
+
+To read more about this deprecation and how to refactor your existing code have a look at [the deprecations page](https://www.emberjs.com/deprecations/v3.x/#toc_object-new-constructor).
+
+**Remove All Listeners/Observers (2 of 6)**
+
+When using both the `removeListener` and `removeObserver` methods, users can omit the final string or method argument to trigger an undocumented codepath that will remove _all_ event listeners/observers for the given key. This functionality will be removed since it is uncommonly used, undocumented, and adds a fair amount of complexity to a critical path.
+
+To read more about this deprecation and how to refactor your existing code have a look at [the deprecations page](https://www.emberjs.com/deprecations/v3.x/#toc_events-remove-all-listeners).
+
+**Deprecate Ember.merge (3 of 6)**
+
+Ever since `Ember.assign` was release `Ember.merge` has been mostly unnecessary. To cut down on duplication, we are now recommending using `Ember.assign` instead of `Ember.merge`.
```

Comment:
```suggestion
Ever since `Ember.assign` was released, `Ember.merge` became mostly unnecessary. To cut down on duplication, we are now recommending using `Ember.assign` instead of `Ember.merge`.
```